cli/js/lib.deno.ns.d.ts

```diff
@@ -1888,8 +1888,17 @@ declare namespace Deno {
    * Requires `allow-run` permission. */
   export function run<T extends RunOptions = RunOptions>(opt: T): Process<T>;
 
-  interface InspectOptions {
+  export interface InspectOptions {
+    /** Traversal depth for nested objects. Defaults to 4. */
     depth?: number;
+    /** Sort Object, Set and Map entries by key. Defaults to false. */
+    sortKeys?: boolean;
+    /** Add a trailing comma for multiline collections. Defaults to false. */
+    trailingComma?: boolean;
+    /** Always break collections into multiple lines. Defaults to false. */
+    alwaysWrap?: boolean;
+    /** The maximum number of iterable entries to print. Defaults to 100. */
+    iterableLimit?: number;
```

Comment:
Are those options named after Node's settings?

Reply:
I thought they only use this for `console.log()`. I'm not aware of any `Deno.inspect()` equivalent in Node, is there one?

Reply:
Oh, found it.

Reply:
Okay, renamed with the exception of `iterableLimit` not being renamed to `maxArrayLength` which would not be correct. Going through their doc history, it's likely legacy: [before](https://nodejs.org/docs/latest-v9.x/api/util.html#util_util_inspect_object_options)/[after](https://nodejs.org/docs/latest-v10.x/api/util.html#util_util_inspect_object_showhidden_depth_colors).

Reply:
Great 👍  thanks for checking that